### PR TITLE
added Pill60 to handwired/pill60

### PR DESCRIPTION
**Pull request by IktaS**
I added Pill60 firmware to handwired folder.

## Description

I added Pill60 firmware to handwired folder.

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/pill60/config.h [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#define VENDOR_ID 0x4454
+#define PRODUCT_ID 0x5444
+#define DEVICE_VER 0x0001
+
+#define MANUFACTURER "IktaS"
+#define PRODUCT "Pill60"
+#define DESCRIPTION "QMK keyboard firmware for Pill60"
```
Comment (drashna) [inline keyboards/handwired/pill60/config.h:9]: This isn't used, and should be removed
```suggestion
```

File: keyboards/handwired/pill60/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,72 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/handwired/pill60/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/handwired/pill60/pill60.h [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#pragma once
```
Comment (drashna) [inline keyboards/handwired/pill60/pill60.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/handwired/pill60/readme.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Pill60
+
+![Pill60](https://i.imgur.com/7WmHa6m.png)
```
Comment (drashna) [inline keyboards/handwired/pill60/readme.md:3]:
```suggestion
![Pill60](https://i.imgur.com/7WmHa6ml.png)
```

**Review by drashna — CHANGES_REQUESTED** [review]
Looks good, minus a couple of things

File: keyboards/handwired/pill60/info.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
+{
+    "keyboard_name": "Pill60",
+    "url": "https://github.com/IktaS/Pill60",
+    "maintainer": "IktaS <imamrafiiia@gmail.com>",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "layout_default": {
+            "key_count": 63,
```
Comment (fauxpark) [inline keyboards/handwired/pill60/info.json:9]:
```suggestion
```

File: keyboards/handwired/pill60/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+        KC_RSFT,   KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_LSFT, MO(_FL), \
+        MO(_FL),  KC_LGUI,  KC_LALT,               KC_SPC,                KC_RALT, KC_RGUI, MO(_FL), KC_RCTL \
+    ),
+    [_FL] = LAYOUT(\
+        KC_TRNS, \
+        KC_GRV,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11, KC_F12, KC_PSCR, KC_INS, \
+        KC_TRNS, KC_TRNS, KC_UP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_UP, KC_TRNS, KC_TRNS,   KC_TRNS, \
+        KC_CAPS,  KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT,    KC_TRNS, \
+        KC_TRNS,  KC_HOME, XXXXXXX, KC_END, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_HOME, XXXXXXX, KC_END,    KC_TRNS, KC_TRNS, \
+        KC_TRNS,  KC_TRNS,  KC_TRNS,                     KC_TRNS,                                KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS \
```
Comment (fauxpark) [inline keyboards/handwired/pill60/keymaps/default/keymap.c:25]:
```suggestion
    [_BL] = LAYOUT(
        KC_F1,
        KC_GESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSLS, KC_DEL,
        KC_TAB,  KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC,   KC_BSPC,
        KC_LCTL,  KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT,    KC_ENT,
        KC_RSFT,   KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH,    KC_LSFT, MO(_FL),
        MO(_FL),  KC_LGUI,  KC_LALT,               KC_SPC,                KC_RALT, KC_RGUI, MO(_FL), KC_RCTL
    ),
    [_FL] = LAYOUT(
        KC_TRNS,
        KC_GRV,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11, KC_F12, KC_PSCR, KC_INS,
        KC_TRNS, KC_TRNS, KC_UP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_UP, KC_TRNS, KC_TRNS,   KC_TRNS,
        KC_CAPS,  KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT,    KC_TRNS,
        KC_TRNS,  KC_HOME, XXXXXXX, KC_END, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_HOME, XXXXXXX, KC_END,    KC_TRNS, KC_TRNS,
        KC_TRNS,  KC_TRNS,  KC_TRNS,                     KC_TRNS,                                KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
```

File: keyboards/handwired/pill60/pill60.h [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+#include "quantum.h"
+#include "led.h"
```
Comment (fauxpark) [inline keyboards/handwired/pill60/pill60.h:4]:
```suggestion
```

File: keyboards/handwired/pill60/blackpill_f401/rules.mk [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+MCU  = cortex-m4
+# ARM version, CORTEX-M0/M1 are 6, CORTEX-M3/M4/M7 are 7
+ARMV = 7
+USE_FPU = yes
+# Address of the booloader in system memory
+STM32_BOOTLOADER_ADDRESS = 0x1FFF0000
+
+# Options to pass to dfu-util when flashing
+DFU_ARGS = -d 0483:df11 -a 0 -s 0x08000000:leave
+DFU_SUFFIX_ARGS = -v 0483 -p df11
```
Comment (fauxpark) [inline keyboards/handwired/pill60/blackpill_f401/rules.mk:27]:
```suggestion
# MCU name
MCU = STM32F401

# Options to pass to dfu-util when flashing
DFU_ARGS = -d 0483:DF11 -a 0 -s 0x08000000:leave
DFU_SUFFIX_ARGS = -v 0483 -p DF11
```

File: keyboards/handwired/pill60/blackpill_f411/rules.mk [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+MCU  = cortex-m4
+# ARM version, CORTEX-M0/M1 are 6, CORTEX-M3/M4/M7 are 7
+ARMV = 7
+USE_FPU = yes
+# Address of the booloader in system memory
+STM32_BOOTLOADER_ADDRESS = 0x1FFF0000
+
+# Options to pass to dfu-util when flashing
+DFU_ARGS = -d 0483:df11 -a 0 -s 0x08000000:leave
+DFU_SUFFIX_ARGS = -v 0483 -p df11
```
Comment (fauxpark) [inline keyboards/handwired/pill60/blackpill_f411/rules.mk:27]:
```suggestion
# MCU name
MCU = STM32F411

# Options to pass to dfu-util when flashing
DFU_ARGS = -d 0483:DF11 -a 0 -s 0x08000000:leave
DFU_SUFFIX_ARGS = -v 0483 -p DF11
```

File: keyboards/handwired/pill60/blackpill_f411/rules.mk [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+# Build Options
+#   change yes to no to disable
+#
+KEYBOARD_SHARED_EP       = yes
```
Comment (fauxpark) [inline keyboards/handwired/pill60/blackpill_f411/rules.mk:32]:
```suggestion
KEYBOARD_SHARED_EP = yes
```

File: keyboards/handwired/pill60/bluepill/bootloader_defs.h [hunk 11]
```diff
@@ -0,0 +1,10 @@
+/* Address for jumping to bootloader on STM32 chips. */
+/* It is chip dependent, the correct number can be looked up here (page 175):
+ * http://www.st.com/web/en/resource/technical/document/application_note/CD00167594.pdf
+ * This also requires a patch to chibios:
+ *  <tmk_dir>/tmk_core/tool/chibios/ch-bootloader-jump.patch
+ */
+
+// STM32F103* does NOT have an USB bootloader in ROM (only serial),
+//  so setting anything here does not make much sense
+#define STM32_BOOTLOADER_ADDRESS 0x80000000
```
Comment (fauxpark) [inline keyboards/handwired/pill60/bluepill/bootloader_defs.h:10]: This file should be deleted.

File: keyboards/handwired/pill60/bluepill/rules.mk [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+MCU = STM32F103
+
+# GENERIC STM32F103C8T6 board - stm32duino bootloader
+OPT_DEFS = -DCORTEX_VTOR_INIT=0x2000
+MCU_LDSCRIPT = STM32F103x8_stm32duino_bootloader
+BOARD = STM32_F103_STM32DUINO
+
+DFU_ARGS = -d 1eaf:0003 -a2 -R
+DFU_SUFFIX_ARGS = -v 1eaf -p 0003
+
```
Comment (fauxpark) [inline keyboards/handwired/pill60/bluepill/rules.mk:11]:
```suggestion
# Bootloader selection
BOOTLOADER = stm32duino
```

File: keyboards/handwired/pill60/blackpill_f401/config.h [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
```
Comment (fauxpark) [inline keyboards/handwired/pill60/blackpill_f401/config.h:20]:
```suggestion
```

File: keyboards/handwired/pill60/config.h [hunk 14]
```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#define VENDOR_ID 0x4454
```
Comment (fauxpark) [inline keyboards/handwired/pill60/config.h:3]:
```suggestion

#include "config_common.h"

#define VENDOR_ID 0x4454
```

File: keyboards/handwired/pill60/bluepill/config.h [hunk 15]
```diff
@@ -0,0 +1,28 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
```
Comment (fauxpark) [inline keyboards/handwired/pill60/bluepill/config.h:20]:
```suggestion
```

File: keyboards/handwired/pill60/blackpill_f411/config.h [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
```
Comment (fauxpark) [inline keyboards/handwired/pill60/blackpill_f411/config.h:20]:
```suggestion
```

File: keyboards/handwired/pill60/rules.mk [hunk 17]
```diff
@@ -0,0 +1,18 @@
...
+
+# Backlight
+BACKLIGHT_ENABLE = yes
+BACKLIGHT_DRIVER = software
+
+#OLED
+OLED_DRIVER_ENABLE = yes
+
+#ENCODER
+ENCODER_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/handwired/pill60/rules.mk:16]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = yes      # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
BACKLIGHT_DRIVER = software
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
OLED_DRIVER_ENABLE = yes
ENCODER_ENABLE = yes
```

File: keyboards/handwired/pill60/keymaps/default/readme.md [hunk 18]
```diff
@@ -0,0 +1,6 @@
+![Clueboard Layout Image](https://i.imgur.com/86mrAwZ.png)
+
```
Comment (fauxpark) [inline keyboards/handwired/pill60/keymaps/default/readme.md:2]:
```suggestion
![Pill60 Layout Image](https://i.imgur.com/86mrAwZ.png)

```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/handwired/pill60/config.h [hunk 19]
```diff
@@ -0,0 +1,38 @@
+#pragma once
```
Comment (drashna) [inline keyboards/handwired/pill60/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**